BasicSample/Assets/GrabAndThrow/Prefabs/SampleSceneUtilities.prefab

```diff
@@ -0,0 +1,745 @@
+%YAML 1.1
```

Comment:
Instead of creating a new prefab here, you can just use the existing SampleSceneUtilities.prefab in Shared/Prefabs and remove this one.